definitions/procedures/packages/enable_modules.rb

```diff
@@ -0,0 +1,18 @@
+module Procedures::Packages
+  class EnableModules < ForemanMaintain::Procedure
+    metadata do
+      description 'Enable the given stream modules'
+      confine do
+        package_manager.class.name == 'Dnf'
+      end
+      param :module_names, 'Module names', :array => true, :required => true
+      param :assumeyes, 'Do not ask for confirmation'
+    end
+
+    def run
+      assumeyes_val = @assumeyes.nil? ? assumeyes? : @assumeyes
+      dnf_option = assumeyes_val ? ' -y' : ''
+      execute!("dnf module enable #{@module_names.join(' ')} #{dnf_option}")
```

Comment:
if we don't pass `-y`, this becomes interactive - is that something that actually works when we spawn `dnf` from inside maintain? like, can the user press "Y" and it is passed to `dnf`?

Reply:
The answer to your question is no, the foreman-maintain does not have capability to prompt.
But yeah, from scenario I am only calling it with assumeyes `true` explicitly, then I also feel should I not keep assumeyes option because it can't be used to false.

Reply:
sounds good!

Reply:
This one is still outstanding.

Reply:
@upadhyeammit Can you fix this last issue?